db/schema.rb

```diff
@@ -204,8 +204,8 @@
 
   create_table "teams", force: true do |t|
     t.string   "name"
-    t.datetime "created_at",      null: false
-    t.datetime "updated_at",      null: false
+    t.datetime "created_at",                      null: false
```

Comment:
The generated in `db/schema.rb` will differ on different machines. We can keep it that way, but sooner or later someone will get a schema.rb that removes the whitespace again. I tend to ignore these changes as they only create noise in the commit log. But as said: let's keep them here :smile: